tempesta_fw/cache.c

```diff
@@ -41,6 +41,11 @@
 #warning "Please set CONFIG_NODES_SHIFT to less than 16"
 #endif
 
+/* Flags stored in a Cache Entry. */
+#define TFW_CE_MUST_REVAL	0x0001		/* MUST revalidate if stale. */
+#define TFW_CE_NO_CACHE		0x0002		/* ALWAYS revalidate. */
```

Comment:
Why do we need the flag? Responses which always must be revalidated shouldn't be stored in cache: if we receive such response then we just bypass the cache.